File: CSharp/DGISoft/DGISoftModel/GrupoInvestigacion.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DGISoftModel
{
    public class GrupoInvestigacion
    {

        private int _idGrupoInvestigacion;
        private BindingList<MiembroPUCP> _integrantes;
        private Seccion _seccion;
        private String _nombre;
        private String _siglas;
        private String _anhoFundacion;
        private bool _interdisciplinario;
        private String _descripcion;
        private byte[] _foto;
        private byte[] _archivoAdicionalPDF;
        private bool _activo;

        public int IdGrupoInvestigacion { get => _idGrupoInvestigacion; set => _idGrupoInvestigacion = value; }
        public Seccion Seccion { get => _seccion; set => _seccion = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public string Siglas { get => _siglas; set => _siglas = value; }
        public string AnhoFundacion { get => _anhoFundacion; set => _anhoFundacion = value; }
        public bool Interdisciplinario { get => _interdisciplinario; set => _interdisciplinario = value; }
        public string Descripcion { get => _descripcion; set => _descripcion = value; }
        public byte[] Foto { get => _foto; set => _foto = value; }
        public byte[] ArchivoAdicionalPDF { get => _archivoAdicionalPDF; set => _archivoAdicionalPDF = value; }
        public bool Activo { get => _activo; set => _activo = value; }
        public BindingList<MiembroPUCP> Integrantes { get => _integrantes; set => _integrantes = value; }
    }
}

File: JAVA/DGISoftModel/src/pe/edu/pucp/dgisoft/model/GrupoInvestigacion.java
package pe.edu.pucp.dgisoft.model;

import java.util.ArrayList;

public class GrupoInvestigacion {
    private int idGrupoInvestigacion;
    private ArrayList<MiembroPUCP> integrantes;
    private Seccion seccion;
    private String nombre;
    private String siglas;
    private String anhoFundacion;
    private boolean interdisciplinario;
    private String descripcion;
    private byte[] foto;
    private byte[] archivoAdicionalPDF;
    private boolean activo;

    public int getIdGrupoInvestigacion() {
        return idGrupoInvestigacion;
    }

    public void setIdGrupoInvestigacion(int idGrupoInvestigacion) {
        this.idGrupoInvestigacion = idGrupoInvestigacion;
    }

    public ArrayList<MiembroPUCP> getIntegrantes() {
        return integrantes;
    }

    public void setIntegrantes(ArrayList<MiembroPUCP> integrantes) {
        this.integrantes = integrantes;
    }

    public Seccion getSeccion() {
        return seccion;
    }

    public void setSeccion(Seccion seccion) {
        this.seccion = seccion;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getSiglas() {
        return siglas;
    }

    public void setSiglas(String siglas) {
        this.siglas = siglas;
    }

    public String getAnhoFundacion() {
        return anhoFundacion;
    }

    public void setAnhoFundacion(String anhoFundacion) {
        this.anhoFundacion = anhoFundacion;
    }

    public boolean isInterdisciplinario() {
        return interdisciplinario;
    }

    public void setInterdisciplinario(boolean interdisciplinario) {
        this.interdisciplinario = interdisciplinario;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public byte[] getFoto() {
        return foto;
    }

    public void setFoto(byte[] foto) {
        this.foto = foto;
    }

    public byte[] getArchivoAdicionalPDF() {
        return archivoAdicionalPDF;
    }

    public void setArchivoAdicionalPDF(byte[] archivoAdicionalPDF) {
        this.archivoAdicionalPDF = archivoAdicionalPDF;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }
    
    
}
